### ISSUE TITLE
Suggestion: Increase contrast

### ISSUE THREAD

**Issue by dwettstein** [sonph/onehalf#110]
I really like the onehalf themes (both light and dark), however, IMHO the contrast could be better, especially when using the light theme.
What do you guys think of having increased contrast?

**Comment by BigBrotherJu** [comment]
Yeah, I totally agree. I'm using OneHalf Light, and the colors are really pleasing to the eye. But the contrasts are very low. It's sometimes hard to differentiate red or green from the white background. However, changing colors to provide more contrast while maintaining the same vibe requires a deep understanding of colors.
